dora/shell/src/main/java/alluxio/cli/bundler/CollectInfo.java

```diff
@@ -394,7 +399,13 @@ private int collectInfoLocal(CommandLine cmdLine) throws IOException {
 
     // Determine the command and working dir path
     String subCommand = args[0];
-    String targetDirPath = args[1];
+    String targetDirPath = "";
+    for (Option opt : cmdLine.getOptions()) {
+      if (opt.equals(OUTPUT_DIR_OPTION)) {
+        targetDirPath = opt.getValue();
+        break;
+      }
+    }
```

Comment:
```suggestion
    String targetDirPath = cmdLine.getOptionValue(OUTPUT_DIR_OPTION_NAME);
```
use `cmdLine.hasOption(OUTPUT_DIR_OPTION_NAME)` or `cmdLine.getOptionValue(OUTPUT_DIR_OPTION_NAME, "")` to handle the inexistence of the option.